account_mass_reconcile_partner/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import advanced_reconciliation
```

Comment:
Can you add the License?

example:
https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py

Reply:
It's not required for __init__.py where there is only import statements.

Reply:
I don't feel that we need a license on import statements.

Reply:
same here :)